File: discover_ruby/github_most_popular.rb
require 'httpclient'
require 'uri'

extheaders = {
  'User-Agent' => 'Holberton_School',
  'Authorization' => 'token f4f6caa6a3e5f5da491e42d6ee14708f325ad655'
}

client = HTTPClient.new
url = "https://api.github.com/search/repositories?q=language:ruby&sort=stars&order=desc"
uri = URI.parse(url)
result = client.get(uri, nil, extheaders)
puts result.content

File: discover_more_python/github_most_popular.py
import requests

resp = requests.get('https://api.github.com/search/repositories?q=language:python&sort=stars&order=desc', headers={
  'User-Agent': 'Holberton_School',
  'Authorization': 'token f4f6caa6a3e5f5da491e42d6ee14708f325ad655'
})

print resp.json()
